.gitignore

```diff
@@ -62,6 +62,4 @@ typings/
 
 .DS_Store
 
-package-lock.json
```

Comment:
not sure if I like this idea, we may want to instruct travis to use the proper eslint version perhaps

Reply:
There's no rush for this PR so I'll see if after a rebase there is some improvement.

Reply:
I rebased the branch to the latest release with the updated dependencies and the travis uses the right `eslint` without needing `package-lock.json`

Reply:
@mattiaerre I'm curious, why don't you like this idea? AFAIK versioning the `package-lock.json` or `yarn.lock` is a pretty standard practice.